pkg/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -232,6 +251,35 @@ func getIgnitionConfig(assetsDir string, profile *performancev2.PerformanceProfi
 			Contents: &rpsService,
 			Name:     getSystemdService("update-rps@"),
 		})
+
+	}
+
+	if profile.Spec.CPU != nil && profile.Spec.CPU.AcceleratedStartup != nil && *profile.Spec.CPU.AcceleratedStartup {
+		script := "accelerated-container-startup"
+		src := filepath.Join(assetsDir, "scripts", fmt.Sprintf("%s.sh", script))
+		mode := 0700
+		scriptPath := getBashScriptPath(script)
+		if err := addFile(ignitionConfig, src, scriptPath, &mode); err != nil {
+			return nil, err
+		}
+		accelerateStartupService, err := getSystemdContent(getAcceleratedStartupUnitOptions(scriptPath))
```

Comment:
Why do we need two services?

Reply:
One for startup and one for shutdown, each with different env variables driving the process.